apps/api/src/config/env-validator.ts

```diff
@@ -30,6 +30,7 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
   NOVU_API_KEY: str({
     default: '',
   }),
+  STORE_ENCRYPTION_KEY: str(),
```

Comment:
@djabarovgeorge is the 32 length required? If so maybe we can add here a min length validation with envalid

Reply:
@scopsy It is a great idea, I did try to create a custom validator for it but I am not sure it supports passing params in it.

Reply:
Actually, let me do one dedicated validator for it.